### PR TITLE
Open Humans connection process update

### PR DESCRIPTION
**Pull request by beaugunderson**
This pull request contains updates for the Open Humans connections process, mostly aimed at providing a better user experience for that process.

It's nearly ready to merge (I'd like to do a bit more local testing) but it's ready for feedback. :sparkles:

### CODE REVIEW

**Comment by wasade** [comment]
I don't see any issues. @squirrelo, can you review too?

File: tox.ini [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [flake8]
 exclude = amgut/lib/locale_data/*,amgut/templates/*,amgut/static/*,amgut/db/*
-max-complexity = 10
\ No newline at end of file
+max-complexity = 10
+
+[pep257]
+ignore = D100,D102,D200,D203,D204,D205,D400
```
Comment (squirrelo) [inline tox.ini]: Not sure if we should be ignoring all these, and if we add this section are we going to then start enforcing pep257 on this repo?

**Comment by squirrelo** [comment]
Code looks good to me. Just have 1 question about pep257 setup.

File: tox.ini (hunk 1 above)
Comment (beaugunderson) [inline tox.ini]: I had my own `.pep257` file in the directory for the code I was adding; I set it to ignore some of the more pedantic errors. `pep257` is unfortunately written in a way that if there's a `tox.ini` file in the directory that it always looks for its settings there and no longer looks in the `.pep257` file.

Decision about whether you all want to use it or not is up to you. :)

**Comment by wasade** [comment]
@squirrelo, if you're :+1:, can you merge?

**Comment by squirrelo** [comment]
Still says it's WIP

**Comment by beaugunderson** [comment]
@squirrelo just removed the WIP, I've tested it on our end as much as I can :)

**Comment by squirrelo** [comment]
Code looks good, so merging.

**Comment by beaugunderson** [comment]
@squirrelo for planning purposes on our end do you have an idea when this will be deployed to production? :)

**Comment by squirrelo** [comment]
Should be now :boom:

**Pull request merged**